example/android/app/build.gradle

```diff
@@ -0,0 +1,126 @@
+apply plugin: 'com.android.application'
+apply plugin: 'jsonschema2dataclass'
+
+android {
+    compileSdkVersion 30
+
+    defaultConfig {
+        applicationId "org.jsonschema2dataclass.android"
+        minSdkVersion 17
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    packagingOptions {
+        exclude 'META-INF/LICENSE.txt'
+    }
+
+}
+
+dependencies {
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+
+    // Required for @Generated annotation
+    implementation 'org.glassfish:javax.annotation:10.0-b28'
+    implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.squareup.moshi:moshi:1.9.2'
+    // Required if generating JSR-303 annotations
+    implementation 'javax.validation:validation-api:1.1.0.CR2'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.5.1'
+
+}
+
+// Each configuration is set to the default value
+jsonschema2dataclass {
```

Comment:
Is this right ?
@eirnym

Reply:
I haven't changed it yet, so `2pojo` here for now